odl/trafos/wavelet.py

```diff
@@ -571,6 +872,11 @@ def __init__(self, dom, nscales, wbasis, mode):
 
             'per': periodization -- like periodic-padding but gives the
             smallest possible number of decomposition coefficients.
+        axes : sequence of `int`, optional
+            Dimensions in which to calculate the wavelet transform.
+            The sequence's length has to be equal to dimension of the ``grid``
```

Comment:
Addition: I saw that you did not add full support for this feature yet. In that case, the documentation is correct. Would be good to have it fully supported anyway.